pkgs/top-level/all-packages.nix

```
@@ -3410,7 +3410,13 @@ with pkgs;
 
   jellycli = callPackage ../applications/audio/jellycli { };
 
-  jellyfin = callPackage ../servers/jellyfin { };
+  jellyfin = callPackage ../servers/jellyfin {
+    ffmpeg = jellyfin-ffmpeg;
+  };
+
+  jellyfin-ffmpeg = callPackage ../development/libraries/jellyfin-ffmpeg {
+    ffmpeg = ffmpeg_4;
+  };
```

```suggestion
  jellyfin = callPackage ../servers/jellyfin {
    ffmpeg = jcallPackage ../development/libraries/jellyfin-ffmpeg { };
  };
```
I think we should make this internal to jellyfin. What do you think?

---

I personally do have a use for `jellyfin-ffmpeg` as a standalone package at the moment. To work around https://github.com/NixOS/nixpkgs/issues/103218, I put it in `environment.systemPackages` so it has a stable path, and then put the path in Jellyfin's stateful settings.

However, to my understanding this has been fixed by https://github.com/jellyfin/jellyfin/pull/6544 and will end up in 10.8.0 (https://github.com/NixOS/nixpkgs/pull/144082) . But Jellyfin 10.8.0 may not make it to NixOS 22.05, as the stable release is still some time away.